src/registries/figures_registry.ts

```diff
@@ -0,0 +1,40 @@
+import { Action } from "../actions/action";
```

Comment:
random rename of this file so that the new file (figure_menu_actions) that contains most of the logic is detected as a MOV by git